rdagent/app/cli.py

```diff
@@ -5,11 +5,14 @@
 - make rdagent a nice entry and
 - autoamtically load dotenv
 """
+from dotenv import load_dotenv
+
+load_dotenv(".env")
```

Comment:
```suggestion
load_dotenv(".env")
# 1) Make sure it is at the beginning of the script so that it will load dotenv before initializing BaseSettings.
# 2) The ".env" argument is necessary to make sure it loads `.env` from the current directory.
```